internal/scorecard/plugins/olm_tests.go

```diff
@@ -165,16 +189,50 @@ func NewOLMTestSuite(conf OLMTestConfig) *schelpers.TestSuite {
 		"Test suite checks if an operator's CSV follows best practices",
 	)
 
+	ts.AddTest(NewBundleValidationTest(conf), 1)
 	ts.AddTest(NewCRDsHaveValidationTest(conf), 1.25)
 	ts.AddTest(NewCRDsHaveResourcesTest(conf), 1)
-	ts.AddTest(NewAnnotationsContainExamplesTest(conf), 1)
+	ts.AddTest(NewCSVValidationTest(conf), 1)
 	ts.AddTest(NewSpecDescriptorsTest(conf), 1)
 	ts.AddTest(NewStatusDescriptorsTest(conf), 1)
 
 	return ts
 }
 
-// Test Implentations
+// Test Implementations
+
+// Run - implements Test interface
+func (t *BundleValidationTest) Run(ctx context.Context) *schelpers.TestResult {
+	res := &schelpers.TestResult{Test: t, MaximumPoints: 1}
+
+	if t.OLMTestConfig.Bundle == "" {
+		res.EarnedPoints++
+		res.Suggestions = append(res.Suggestions, "Add a 'bundle' directory which is required for this test")
```

Comment:
This seems more like an Error than Suggestion.

Reply:
agree, I've changed it to produce an Error instead of a Suggestion.